css/style.css

```diff
@@ -221,7 +233,11 @@ header {
     font-size: 50px;
     font-weight: bold;
     backdrop-filter: blur(20px);
-}
+    width: 100%;
+    color: turquoise;
```

Comment:
Instead of named color use the hexcode so that the range of browser support is more.